### PR TITLE
Desktop Version

### PR DESCRIPTION
**Pull request by 1GORDON**
# Desktop Version Media Query
* Implemented responsiveness

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,24 +8,29 @@
...
         <section class="main-section">
             <br>
-            <h1 class="main-text">I’m Godanah
+            <h1 class="main-text">I’m Godanah <br>
                 Glad to see you!</h1>
             <p class="content">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
             <br>
-            <p>LET'S CONNECT</p>
+            <p class="p-page1">LET'S CONNECT</p>
             <br>
```
Comment (Emmyn5600) [inline index.html:34]: - [x] Kindly, let us readjust the padding-top for our heading to get the same we have on the figma
port43

File: index.html [hunk 2]
```diff
@@ -81,23 +117,54 @@ <h1>Multi-Post<br>Stories</h1>
...
-            <p class="contents">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <p class="content">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
             <br>
             <div class="buttons">
                 <ul class="list">
                     <li><div class="tag">html</div></li>
                     <li><div class="tag">css</div></li>
                     <li><div class="tag">javascript</div></li>
                 </ul>
+                <button class="active">See Project</button>
```
Comment (Emmyn5600) [inline index.html:128]: - [x] Kindly change the color for the `see project` button to get similar colors with see project buttons
port44

File: index.html [hunk 3]
```diff
@@ -192,14 +320,78 @@ <h1 class="main-text">About Myself</h1>
...
+                                <a href="#"><img class="lang-logo" src="images/css-logo.png" alt="css"></a>
+                                <p class="upper">css</p>
+                            </li>
+                        </ul>
+                    </li>
+                    <li class="coder-li"><a href="#">Frameworks</a><img class="coder-img" src="images/chevron.png" alt="frameworks"></li>
+                    <li class="coder-li"><a href="#">Skills</a><img class="coder-img" src="images/chevron.png" alt="skills"></li>
+                </ul>
+            </div>
+        </div>
```
Comment (Emmyn5600) [inline index.html:369]: - [x] Kindly work on the techbnlogies (you need to change the font-size) as well as arrow on each technology, refer to the figma for more details:

port45

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @1GORDON 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
Changes required :writing_hand:  
      * #### Kindly, Add the descriptive PR for your porject.

      * #### Please check the comment under the review
       
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 2 above)
Comment (1GORDON) [reply index.html:128]: Hello Emmy.  Thank you for the guidance. It is highly appreciated. However, the color of the third card is actually like that on the Figma project

File: index.html (hunk 3 above)
Comment (1GORDON) [reply index.html:369]: Thank you for the review. Fixed

File: index.html (hunk 1 above)
Comment (1GORDON) [reply index.html:34]: Fixed. Thanks for the review

**Comment by 1GORDON** [comment]
I have just realized that I merged the branch with the main project. This was because we had another project of animations that we had to merge with the main branch. I think that is how I mistakenly merged this branch with the main branch. However, I have implemented the changes and pushed them to the main. My bad about that. Thank you for your guidance again

File: index.html [hunk 4]
```diff
@@ -192,14 +320,78 @@ <h1 class="main-text">About Myself</h1>
...
+    </section>
```
Comment (SajjadAhmad14) [inline index.html:370]: Languages, Skills, and Frameworks are not images. Please see the requirements again and make them a list.

File: index.html [hunk 5]
```diff
@@ -8,24 +8,29 @@
     <title>my-portfolio</title>
 </head>
 <body>
```
Comment (SajjadAhmad14) [inline index.html:10]: Please do not merge PR without approval from a reviewer.

Comment (SajjadAhmad14) [inline index.html:10]: [ OPTIONAL ] There is a horizontal scroll on the page for medium and small viewports which shows that some content is overflowing from the container. Please remove the horizontal scroll.

**Review by SajjadAhmad14 — COMMENTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Godanah Tinyefuza, 
 ### Thank you for working on changes suggested :ok_hand:

## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by 1GORDON** [comment]
It seems I cannot re-open a merged pull request. I am stuck. I don't know what to do next. Any guidance will be appreciated

File: index.html (hunk 5 above)
Comment (1GORDON) [reply index.html:10]: It seems I can't reopen a merged pull request. Any guidance will be appreciated

File: index.html (hunk 4 above)
Comment (1GORDON) [reply index.html:370]: Thank you again. Yes, Languages Skills and Frameworks are not images. I already implemented that

File: index.html (hunk 5 above)
Comment (1GORDON) [reply index.html:10]: Thank you so much. This is also fxed

**Pull request merged**